homeassistant/components/here_travel_time/config_flow.py

```diff
@@ -88,13 +91,16 @@ def get_user_step_schema(data: dict[str, Any]) -> vol.Schema:
 
 def default_options(hass: HomeAssistant) -> dict[str, str | None]:
     """Get the default options."""
-    return {
+    default = {
         CONF_TRAFFIC_MODE: TRAFFIC_MODE_ENABLED,
         CONF_ROUTE_MODE: ROUTE_MODE_FASTEST,
         CONF_ARRIVAL_TIME: None,
         CONF_DEPARTURE_TIME: None,
-        CONF_UNIT_SYSTEM: hass.config.units.name,
+        CONF_UNIT_SYSTEM: CONF_UNIT_SYSTEM_METRIC,
```

Comment:
Checking/Storing the `name` of the unit system will stop working if we rename or add a new unit system.
It is more future-proof to check the `length_unit` of the unit system, even if the component still uses "imperial" terminology.

Reply:
Adjusted to use `hass.config.units == ***`